File: 프로그래머스/0/120831. 짝수의 합/짝수의 합.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int n) {
    int sumv = 0;
    for (int i = 1; i <= n; i++){
        if (i % 2 == 0) {
            sumv += i;
        }
    }
    return sumv;
}

File: 프로그래머스/0/120831. 짝수의 합/짝수의 합.py
def solution(n):
    if n == 0:
        return 0
    
    if  2<= n <= 3:
        return 2
    
    n = n / 2 if n % 2== 0 else (n - 1) / 2
    
    return n**2 + n
    